libleidenalgConfig.cmake.in

```diff
@@ -1,21 +1,22 @@
 # - Config file for the libleidenalg package
 # It defines the following variables
-#  LIBLEIDENALG_INCLUDE_DIRS - include directories for libleidenalg
+#  LIBLEIDENALG_INCLUDEDIR - include directories for libleidenalg
 #  LIBLEIDENALG_LIBRARIES    - libraries to link against
 
-set(LIBLEIDENALG_VERSION 0.9.0)
+set(LIBLEIDENALG_VERSION "@PACKAGE_VERSION_BASE@")
+  if (CMAKE_BUILD_TYPE EQUAL "RELEASE")
```

Comment:
Shouldn't these things be set also for Debug builds by the way?

Reply:
For sure-oversight on my part.

Reply:
wait-i may have read that wrong. do you mean the version? or do you mean everything below this?

Reply:
I mean that I'm not sure why it would be necessary to limiting everything below to a release build?

Reply:
If I'm remembering correctly-as it was a while ago-the reason I did this was because vcpkg was unhappy about the exported targets file also being present in the debug build. If you want the debug build to create the exported targets that's fine.